src/app/colors/colorizer-data.model.ts

```diff
@@ -10,7 +10,7 @@ export interface DeprecatedMappingColorizerDoNotUse {
     result?: string | number;
 }
 
-export type ColorizerType = 'gradient' | 'palette';
+export type ColorizerType = 'gradient' | 'palette' | 'rgba';
```

Comment:
'gradient' and 'palette' also create rgb colors.

Reply:
see https://github.com/umr-dbs/wave/pull/12/commits/728ca63d51771dc5b52bd260da6c4c402a6b3b50